diff --git a/depth_anything_v2/dpt.py b/depth_anything_v2/dpt.py
index 18d3e6f..692fa83 100644
--- a/depth_anything_v2/dpt.py
+++ b/depth_anything_v2/dpt.py
@@ -1 +0,0 @@
-import cv2
@@ -5 +3,0 @@ import torch.nn.functional as F
-from torchvision.transforms import Compose
@@ -9 +6,0 @@ from .util.blocks import FeatureFusionBlock, _make_scratch
-from .util.transform import Resize, NormalizeImage, PrepareForNet
@@ -124,2 +120,0 @@ class DPTHead(nn.Module):
-            else:
-                x = x[0]
@@ -171,0 +167 @@ class DepthAnythingV2(nn.Module):
+        self.use_clstoken = use_clstoken
@@ -179 +175,5 @@ class DepthAnythingV2(nn.Module):
-        features = self.pretrained.get_intermediate_layers(x, self.intermediate_layer_idx[self.encoder], return_class_token=True)
+        features = self.pretrained.get_intermediate_layers(
+            x,
+            self.intermediate_layer_idx[self.encoder],
+            return_class_token=self.use_clstoken,
+        )
@@ -182 +181,0 @@ class DepthAnythingV2(nn.Module):
-        depth = F.relu(depth)
@@ -196,0 +196,5 @@ class DepthAnythingV2(nn.Module):
+        import cv2
+        from torchvision.transforms import Compose
+
+        from .util.transform import Resize, NormalizeImage, PrepareForNet
+
